pkg/apis/certmanager/v1/generic_issuer.go

```diff
@@ -35,43 +36,67 @@ type GenericIssuer interface {
 var _ GenericIssuer = &Issuer{}
 var _ GenericIssuer = &ClusterIssuer{}
 
+// GetObjectMeta is ...
 func (c *ClusterIssuer) GetObjectMeta() *metav1.ObjectMeta {
 	return &c.ObjectMeta
 }
+
+// GetSpec is ...
 func (c *ClusterIssuer) GetSpec() *IssuerSpec {
 	return &c.Spec
 }
+
+// GetStatus is ...
 func (c *ClusterIssuer) GetStatus() *IssuerStatus {
 	return &c.Status
 }
+
+// SetSpec is ...
 func (c *ClusterIssuer) SetSpec(spec IssuerSpec) {
 	c.Spec = spec
 }
+
+// SetStatus is ...
 func (c *ClusterIssuer) SetStatus(status IssuerStatus) {
 	c.Status = status
 }
+
+// Copy is ...
 func (c *ClusterIssuer) Copy() GenericIssuer {
 	return c.DeepCopy()
 }
+
+// GetObjectMeta is ...
 func (c *Issuer) GetObjectMeta() *metav1.ObjectMeta {
 	return &c.ObjectMeta
 }
+
+// GetSpec is ...
 func (c *Issuer) GetSpec() *IssuerSpec {
 	return &c.Spec
 }
+
+// GetStatus is ...
 func (c *Issuer) GetStatus() *IssuerStatus {
 	return &c.Status
 }
+
+// SetSpec is ...
 func (c *Issuer) SetSpec(spec IssuerSpec) {
 	c.Spec = spec
 }
+
+// SetStatus is ...
 func (c *Issuer) SetStatus(status IssuerStatus) {
 	c.Status = status
 }
+
+// Copy is ...
```

Comment:
Is this a good pattern?
```suggestion
// Copy returns a Deep copy of `GenericIssuer`
```